src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person | undefined;
```

Comment:
```suggestion
  person: Person;
```
you render this component only when person exists